frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.ts

```diff
@@ -307,6 +313,55 @@ export class TeamPlannerComponent extends UntilDestroyedMixin implements OnInit,
 
   isMobile = this.deviceService.isMobile;
 
+  private initialCalendarView = this.workPackagesCalendar.initialView || 'resourceTimelineWorkWeek';
+
+  private viewOptionDefaults = {
+    type: 'resourceTimeline',
+    slotDuration: { days: 1 },
+    resourceAreaColumns: [
+      {
+        field: 'title',
+        headerContent: {
+          html: `<span class="spot-icon spot-icon_user"></span> <span class="hidden-for-mobile">${this.text.assignee}</span>`,
```

Comment:
might be good to have a `aria-label` on the icon with the assignee text so mobile screenreaders still have text to read

Reply:
Done in [cfb9569](https://github.com/opf/openproject/pull/11601/commits/cfb9569e5b140a20c76650f895d9631f77044a6c)